src/modules/Builder/components/MediaUploader/MediaUploader.tsx

```diff
@@ -1,83 +1,84 @@
 import { Trans, useTranslation } from 'react-i18next';
 
-import { StyledBodyMedium, StyledTitleTooltipIcon, theme, variables } from 'shared/styles';
-import { Svg, Tooltip } from 'shared/components';
+import {
+  StyledBodyMedium,
+  StyledFlexColumn,
+  StyledLinearProgress,
+  StyledLabelBoldLarge,
+  theme,
+  variables,
+} from 'shared/styles';
+import { Svg } from 'shared/components';
 import { byteFormatter } from 'shared/utils';
 import { MAX_FILE_SIZE_8MB } from 'shared/consts';
 
-import {
-  StyledContainer,
-  StyledNameWrapper,
-  StyledSourceContainer,
-  StyledTitle,
-} from './MediaUploader.styles';
+import { StyledContainer, StyledSourceContainer, StyledPreview } from './MediaUploader.styles';
 import { useMediaUploader } from './MediaUploader.hooks';
 import { MediaUploaderProps } from './MediaUploader.types';
 import { MLPlayer } from '../MLPlayer';
 
 export const MediaUploader = ({
   width,
   height,
-  resourceData,
-  setResourceData,
+  media,
+  placeholder,
+  hasPreview,
+  onUpload,
 }: MediaUploaderProps) => {
   const { t } = useTranslation('app');
   const { uploadInputRef, error, dragEvents, handleChange, onRemove } = useMediaUploader({
-    setResourceData,
+    onUpload,
   });
 
   return (
     <>
       <input
         ref={uploadInputRef}
         onChange={handleChange}
-        accept="audio/*"
+        accept=".mp3,.wav"
```

Comment:
Looks like wysiwyg will use the same file types, can you move this value in builder constants?

Reply:
done